packages/flutter/lib/src/material/progress_indicator_theme.dart

```diff
@@ -0,0 +1,170 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'theme.dart';
+
+@immutable
+/// Defines the visual properties of [ProgressIndicator] widgets.
+///
+/// Used by [ProgressIndicatorTheme] to control the visual properties of
+/// progress indicators in a widget subtree.
+///
+/// To obtain this configuration, use [ProgressIndicatorTheme.of] to access
+/// the closest ancestor [ProgressIndicatorTheme] of the current [BuildContext].
+///
+/// See also:
+///
+///  * [ProgressIndicatorTheme], an [InheritedWidget] that propagates the
+///    theme down its subtree.
+///  * [ThemeData.progressIndicatorTheme], which describes the defaults for
+///    any progress indicators as part of the application's [ThemeData].
+class ProgressIndicatorThemeData with Diagnosticable {
+  /// Creates the set of properties used to configure [ProgressIndicator] widgets.
+  const ProgressIndicatorThemeData({
+    this.backgroundColor,
+    this.color,
+  });
+
+  /// The color of the [ProgressIndicator]'s track.
+  ///
+  /// This color is interpreted differently depending on the type of
+  /// indicator:
+  ///
```

Comment:
If we're just reiterating the API doc for these properties then using something like
```
 /// {@macro flutter.material.LinearProgressIndicator.backgroundColor}
```
would be mean less chance of things getting out of sync.

Reply:
Yup, I moved everything to macros to remove this duplication.